src/main/java/com/moviecat/servicesImpl/genre/GenreServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+/*
+ *  Licensed to the MovieCat, Inc. under one
+ *  or more contributor license agreements. See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.
+ *
+ *  You may not use this file except in compliance
+ *  with the License.
+ *
+ */
+
+package com.moviecat.servicesImpl.genre;
+
+import com.moviecat.ds.DbProvider;
+import com.moviecat.model.Genre;
+import com.moviecat.services.GenreServices;
+import com.moviecat.servicesImpl.implementations.BaseService;
+
+import dev.morphia.query.Query;
+import dev.morphia.query.UpdateOperations;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class GenreServiceImpl implements GenreServices {
+
+    private static final Logger log = LogManager.getLogger(GenreServiceImpl.class);
+    BaseService service = new BaseService();
```

Comment:
Skip creating an instance of BaseService.
All methods in BaseService are static, and can be referenced directly. e.g. `BaseService.fetch()`

Reply:
Great.